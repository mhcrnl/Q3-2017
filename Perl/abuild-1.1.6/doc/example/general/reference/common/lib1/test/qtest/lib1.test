#!/usr/bin/env perl

require 5.008;
BEGIN { $^W = 1; }
use strict;

chdir("lib1") or die "chdir testdir failed: $!\n";

require TestDriver;

my $td = new TestDriver('lib1');

$td->runtest("test lib1 class",
	     {$td->COMMAND => "lib1_test"},
	     {$td->FILE => "lib1.out", $td->EXIT_STATUS => 0},
	     $td->NORMALIZE_NEWLINES);

$td->report(1);
