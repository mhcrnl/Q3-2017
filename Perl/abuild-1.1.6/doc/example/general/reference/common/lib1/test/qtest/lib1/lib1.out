not counting backwards from -12
counting backwards from 8: 8 7 6 5 4 3 2 1
